flank-scripts/src/main/kotlin/flank/scripts/exceptions/FlankScriptsExceptionMappers.kt

```diff
@@ -10,6 +10,8 @@ fun <V : Any, E : FuelError, E2 : Exception> Result<V, E>.mapClientError(transfo
     is Result.Failure -> if (error.response.isClientError) Result.Failure(transform(error)) else this
 }
 
+fun <V : Any, E : FuelError> Result<V, E>.mapErrorToGithubException() = mapClientError { it.toGithubException() }
```

Comment:
Are you opposed to actually labeling the V & E to actual names for example:
```suggestion
fun <VALUE : Any, ERROR : FuelError> Result<VALUE , ERROR >.mapErrorToGithubException() = mapClientError { it.toGithubException() }
```
It does nothing but increase readability

Reply:
Sure, I'll change it. I kept convention from the other functions. I'll make this change there as well